src/utils/check-icons.tsx

```diff
@@ -0,0 +1,13 @@
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline'
+import DriveFileRenameOutlineOutlinedIcon from '@mui/icons-material/DriveFileRenameOutlineOutlined'
+import FormatListBulletedOutlinedIcon from '@mui/icons-material/FormatListBulletedOutlined'
```

Comment:
can destructure imports from '@mui/icons-material'

Reply:
> can destructure imports from '@mui/icons-material'

i think we can leave it like this, we need to practice treeshaking, so this implementation seems fine